analysis/periscope/models_out_of_sla.sql

```diff
@@ -0,0 +1,17 @@
+with
+  models_out_of_sla as (
+    select
+      model
+      , max(deployment_started_at) as last_deployment_ts
+      , datediff('hour', max(deployment_started_at), getdate()) as hours_since_refreshed
```

Comment:
I'm not sure what db you're on but isn't `CURRENT_DATE` more universal? I had to make that change here for Snowflake.

Reply:
This will be multi-db compatible
```suggestion
      , datediff('hour', max(deployment_started_at), {{ dbt_utils.current_timestamp() }}) as hours_since_refreshed
```